src/source-napatech.c

```diff
@@ -359,6 +359,35 @@ static int CompareIPv6Addr(uint8_t addr_a[16], uint8_t addr_b[16]) {
     return 0;
 }
 
+/**
+ * \brief  Initializes the FlowStreams used to program flow data.
+ *
+ * Opens a FlowStream on the adapter associated with the rx port.  This
+ * FlowStream is subsequently used to program the adapter with
+ * flows to bypass.
+ *
+ * \return the flow stream handle, NULL if failure.
+ */
+static NtFlowStream_t InitFlowStream(int adapter, int stream_id)
+{
+    int status;
+    NtFlowStream_t hFlowStream;
+
+        NtFlowAttr_t attr;
```

Comment:
This needs to be fixed, as pointed out 2(!) times already.

Reply:
I apologize.  I remember making this change.  But, I had a git glitch and had to redo a bunch of changes.  Apparently, I missed this one.